Assets/Scripts/ActionMaster.cs

```diff
@@ -1,48 +0,0 @@
-﻿using System;
-using UnityEngine;
-using System.Collections;
-using System.Collections.Generic;
-using System.Linq;
-using System.Resources;
-
-public class ActionMaster
-{
-	
-	public enum Action
```

Comment:
Good making on this enum, some following old naming conventions from other languages prefix enums, e.g. "EAction" but that isn't the convention in C#